chingu-voyages/v19-bears-team-07
# View manage apps

Set up new backend endpoints to fetch apps that *specifically belong to* the authenticated user.

Implemented a frontend at `/manage-apps` that will display the authenticated user's apps.

To test:

- Set up the db with the seed data
- Log in with one of the seed user accounts (you can find the credentials in `seeds.rb`
- Visit `/manage-apps` to see just the apps for this user account
- Repeat with a different seed user account to verify that different apps are shown for a different user.
